nbextensions/README.md

```diff
@@ -0,0 +1,4 @@
+# Jupyter notebook extensions
+
+This directory contains the Jupyter notebook extensions
+that are useful for the programming education assistant.
```

Comment:
Nice to have a link about Jupyter notebook extension development guide.
maybe this?
https://jupyter-notebook.readthedocs.io/en/stable/extending/frontend_extensions.html

Reply:
Nice idea, thanks! Added the link.